### ISSUE TITLE
Descriptions based on settings

### ISSUE THREAD

**Issue by tubbynl** [humanswitch/consentcookie#6]
## Description
Some details in the description need to be based on tracker settings on the website

## Expected Behavior
The description "Google Analytics verzameld geanonimiseerd gegevens over het surfgedrag op deze website. Door het anonimiseren is dit niet naar jou te herleiden." should partially be based on the initstate setting.

"Door het anonimiseren is dit niet naar jou te herleiden." depends on how GA is configured (anonymise thingie).

## Possible Fix
This could reflect as a option for 'initstate'
* disabled: Om dit mogelijk te maken staat {} altijd aan.
* optin: Je bepaalt zelf of je {} aan- of uitzet.
* optout: Je bepaalt zelf of je {} uit- of aanzet.
* anonimised:  Door anonimisering is dit niet naar jou te herleiden en staat {} altijd aan.

**Comment by StevenChoo** [comment]
Need to discuss it.

Several questions:
* is it a common setting "anonymised" of is it specific for 1 (or 2) trackers
* if it's not common, how will we handle options that are tracker specific (in general)
* how do we guarantee that if it's set anonymised the trackers works like that (and is this a relevant issue). Especially with the current Facebook debate and companies collecting data.

**Comment by tubbynl** [comment]
no, i think it should be a setting like "you can't disable it and i(webmaster) have set the settings of this tracker in such a way it's workings are anonimised"

being that the webmaster "says" he's done actions in the trackers config to ensure anonimity

**Comment by StevenChoo** [comment]
I think one of the problems is that the state currently is called: disabled.

Something like "alwayson" would be more clear.

With that, adding a flag next to the ```initstate``` property, e.g. ```"anomymous":"true"``` would give the option to let ConsentCookie know the configured tracker/app is collecting data anonymously.

connection config example:
```json
{
	"initstate": 1,
	"anonymous": true
}
```

With this you can also provide an option for optin / optout and set the flag anomymous = true
In the "connections" if the flag is set, ConsentCookie will show a generic message e.g
```"This website uses this tracker to collect data anonymously" ```

(the used naming are a proposal and for debate)

**Comment by tubbynl** [comment]
yes, that is more explicit and transparent

**Comment by StevenChoo** [comment]
> i assum the enum-values of initState remains are as they are used now (optin-optout-disabled)

Yes

optin = 1
optout = 0
alwayson = -1

**Comment by StevenChoo** [comment]
partially fixed by #15
